### DISCUSSION TITLE
build duckdb failed with DBUILD_ARROW_ABI_TEST in Mac M1

### DISCUSSION THREAD

**Discussion by huxiaoxiang** [duckdb/duckdb#5726] [General]
Hi, I encountered a problem when I turn DBUILD_ARROW_ABI_TEST on.

How to reproduce:
- Branch: master
- Build Cmake Options: `-DCMAKE_BUILD_TYPE=Debug -DBUILD_ARROW_ABI_TEST=TRUE -DBUILD_UNITTESTS=TRUE`
- Machine: MacOS 11.3.1， Apple M1

Here are some suspections in building logs:
![image](https://user-images.githubusercontent.com/4969695/208284158-8edef405-81ad-45a4-9f14-afdd7ecad67a.png)
![image](https://user-images.githubusercontent.com/4969695/208284179-7b76fad2-07f6-4b52-a334-ed1a7013805c.png)
![image](https://user-images.githubusercontent.com/4969695/208284184-b3a03e58-b6a3-41e8-872f-64322004dae1.png)
![image](https://user-images.githubusercontent.com/4969695/208284193-97486f10-0b97-4ca6-833d-0a9ba58e882f.png)
![image](https://user-images.githubusercontent.com/4969695/208284242-58495dd1-817d-4008-83f9-00f8ab304efc.png)

Any help is appreciated!

**Comment by huxiaoxiang** [comment]
Would someone help me...

**Reply by Mytherin** [reply]
Why are you building the arrow API tests in the first place?

**Reply by huxiaoxiang** [reply]
> Why are you building the arrow API tests in the first place?

Well, I tried:
-DCMAKE_BUILD_TYPE=Debug -DBUILD_UNITTESTS=TRUE         // no problem
-DCMAKE_BUILD_TYPE=Debug -DBUILD_UNITTESTS=TRUE -DBUILD_ARROW_ABI_TEST=TRUE     // problems

I'll figure out if there're configuration problems with arrow in third_party. 
Thanks!
